pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -121,7 +121,7 @@ test: manifests generate fmt vet envtest ## Run tests.
 ##@ Build
 
 .PHONY: build
-build: generate fmt vet ## Build manager binary.
+build: manifests generate fmt vet ## Build manager binary.
```

Comment:
It seems a nice change.
We would need to do that for go/v3 and go/v4-alpha plugins

Could you please create a PR for this one?

Reply:
Got it, raised a new PR: https://github.com/kubernetes-sigs/kubebuilder/pull/3020
PTAL thanks.